extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-preview.css

```diff
@@ -0,0 +1,259 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ amp-story-dev-tools-tab-preview {
+  --i-amphtml-story-dev-tools-device-shadow: 0px 11.0046px 27.5114px rgba(0, 0, 0, 0.34), inset 0px 2.20091px 3.30137px rgba(44, 46, 52, 0.6) !important;
+
+  --i-amphtml-story-dev-tools-device-gray-1: #101010 !important;
+  --i-amphtml-story-dev-tools-device-gray-2: #2E3137 !important;
+  --i-amphtml-story-dev-tools-device-gray-3: #2E2E2E !important;
+  --i-amphtml-story-dev-tools-device-gray-4: #3D3D3D !important;
+  --i-amphtml-story-dev-tools-device-gray-transparent-1: #111111CC !important;
+  --i-amphtml-story-dev-tools-device-white-transparent-1: #FFFFFF22 !important;
+  --i-amphtml-story-dev-tools-device-white-transparent-2: #FFFFFF77 !important;
+  --i-amphtml-story-dev-tools-device-white-transparent-3: #FFFFFFCC !important;
+
+  --i-amphtml-story-dev-tools-material-easing: cubic-bezier(0.4, 0.0, 0.2, 1);
+  --i-amphtml-story-dev-tools-material-easing-decelerated: cubic-bezier(0.0, 0.0, 0.2, 1);
+ }
+
+.i-amphtml-story-dev-tools-device {
+  background: var(--i-amphtml-story-dev-tools-device-gray-1) !important;
+  border-radius: 40px !important;
+  box-shadow: var(--i-amphtml-story-dev-tools-device-shadow) !important;
+  overflow: hidden !important;
```

Comment:
Is this needed?
It looks like the width of this element is determined by its content.

Reply:
We need it because the iframes need to be "cut" by the device border-radius.

Eg on desktop, but will happen in other devices as well:

![image](https://user-images.githubusercontent.com/22420856/100923928-efacd500-34ad-11eb-89a5-a6652c3b354d.png)